filters/filters/PhotoEnhancer/PhotoEnhancer/Filters/NoiseFilter.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace PhotoEnhancer
+{
+    public class NoiseFilter:PixelFilter
+    {
+        public NoiseFilter() : base(new NoiseParameters()) { }
+        public override Pixel ProcessPixel(Pixel original, IParameters parameters)
+        {
+            Random rnd = new Random();
+            double k =(parameters as NoiseParameters).Coefficent;
```

Comment:
Фильтр работает неправильно. Нельзя каждый раз создавать новый ГСЧ. Сделайте его статическим полем класса, чтобы он создавался только один раз, и всё заработает как надо
```suggestion
    public class NoiseFilter:PixelFilter
    {
        static Random rnd = new Random();
        public NoiseFilter() : base(new NoiseParameters()) { }
        public override Pixel ProcessPixel(Pixel original, IParameters parameters)
        {
            double k =(parameters as NoiseParameters).Coefficent;
```